bin/variants2fasta.py

```diff
@@ -7,13 +7,21 @@
 from Fred2.Core.Variant import VariationType
```

Comment:
Ah here and in other scripts it would make sense to port the scripts to `Epytope` but probably a task for the next release 😄

Reply:
This is indeed next on the list 👍